dash_shared/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module allows to
```

Comment:
"Aaargh!!"
( https://www.youtube.com/watch?v=ZlIz0q8aWpA )

Reply:
Good point !  I did it but not on the PR... I will push a commit in 14.0 branch